marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -139,10 +140,11 @@ instance FromJSON Utxo where
             <*> v .: "value"
             <*> v .: "inlineScript"
             <*> v .: "inlineScriptHash"
+            <*> v .: "blockIndex"
     parseJSON _ = mempty
 
 instance ToJSON Utxo where
-  toJSON (Utxo addr txid txix dtum dtumHash val scrpt scrptHash) = object
+  toJSON (Utxo addr txid txix dtum dtumHash val scrpt scrptHash blockIndex') = object
```

Comment:
Same here. I'll stop repeating, but any mention of `blockindex` should be `txIndexInBlock`